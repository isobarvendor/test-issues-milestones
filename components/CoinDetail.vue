<template>
    <div class="coin-detail">
        <h3> Enter Code </h3>
            <div class="input-container">
                <v-app id="inspire" style="">
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                    <v-text-field color="white"></v-text-field>
                </v-app>
            </div>
            <div class="button-container">
                <a  class="button">Collect</a>
            </div>
    </div>
</template>

<script>
export default {
  name: "CoinDetail",
  components: {
  },
  data() {
    return {
    };
  },
  props: {
    data: null,
  },
  computed:{
  },
  methods:{
  },
  beforeMount() {},
  mounted(){

  }
};
</script>

<style lang="scss">
.coin-detail{
    display: flex;
    align-items: center;
    flex-direction: column;
    .theme--light.v-application{
        background: none !important;
    }
    .v-application--wrap{
        min-height:auto !important;
        display: flex;
        flex-direction: row;
    }
    .v-application .primary--text{
        caret-color: #ffffff !important;
    }
    .theme--light.v-input{
        width: 25px;
        margin-right: 15px;
        &:last-child{
            margin-right: 0px;
        }

    }
    
    .theme--light.v-input input{
        color: white;
        text-align: center;
    }
    .theme--light.v-text-field > .v-input__control > .v-input__slot:before{
        border-color: #585858;
    }
    .mdi:before, .mdi-set{
        color: white;
    }
    .input-container{
        margin-top: 40px
    }
    @media only screen and (max-width: 1199px) {
          
    }  
}
</style>
