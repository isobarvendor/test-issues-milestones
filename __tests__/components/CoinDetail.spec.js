
import { shallowMount } from '@vue/test-utils'

import CoinDetail from "@/components/CoinDetail";


const factory = () => {
  return shallowMount(CoinDetail);
};

describe("CoinDetail", () => {


  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.vm).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
