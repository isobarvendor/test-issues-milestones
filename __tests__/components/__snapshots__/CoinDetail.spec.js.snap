// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoinDetail renders properly 1`] = `
<div class="coin-detail">
  <h3> Enter Code </h3>
  <div class="input-container">
    <v-app-stub id="inspire">
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
      <v-text-field-stub color="white" errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
    </v-app-stub>
  </div>
  <div class="button-container"><a class="button">Collect</a></div>
</div>
`;
